lib/multiply.rb

```diff
@@ -1,2 +1,17 @@
 def multiply(num1, num2)
+    
+new_arr = Array.new
+
+#Included the if statement in the case where there would be a large disparity between numbers
+if (num1 > num2) || (num1 - num2) > 20
```

Comment:
Just something I thought could add efficiency. So for instance, if the numbers provided are 20 and 9, I would want 20 to be the number listed since there's fewer of them. In extreme cases like 150  and 6, I thought this could be a big benefit. So, I added the conditional to look for the first number being the biggest number, or if there's a difference of at least 20 between the two numbers (just an arbitrary threshold I used).